StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -48,6 +48,27 @@ extension Client {
                                                           extraData: extraData)
         return .init(channelQuery: .init(channelPayload: payload), client: self)
     }
+
+    /// Creates a new `ChannelController` that will create new channel with members without id. It's great for direct message channels.
+    /// - Note: The number of members should be more than 1.
+    /// - Parameters:
+    ///   - members: Members for the new channel.
+    ///   - type: Type of the new channel.
+    ///   - team: Team for the new channel.
+    ///   - extraData: Extra data for the new channel.
+    /// - Returns: A new instance of `ChannelController`.
+    public func channelController(with members: Set<UserId>,
+                                  type: ChannelType = .messaging,
+                                  team: String? = nil,
+                                  extraData: ExtraData.Channel
+    ) -> ChannelControllerGeneric<ExtraData> {
+        let payload = ChannelEditDetailPayload<ExtraData>(cid: .init(type: type, id: ""),
```

Comment:
It's no longer possible to create a channel without id locally, because of the local DB. We need to create the id here. Ideally, you'd create a helper `ChannelId` extension for this, that would create it. One possibility would be to get members' ids, sort them, and somehow create a has from it.

This should be tested too. Especially for edge cases like `[A, B] == [B, A]`, many members, one member, etc...

Reply:
@mantergo please ignore this comment for now. I need to verify something.